### PR TITLE
fix(lint): clean up the eslint config

### PR DESCRIPTION
**Pull request by moodysalem**
we had eslint config in two different places, which seemed to cause the `react-app` config to be ignored 

also the lint action still fails on prs, so this fixes the action to just run `yarn eslint .` on forks and the setup-node step sets up matchers to read the output of the linter to create annotations

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -38,10 +38,15 @@ jobs:
...
         with:
           github_token: ${{ secrets.github_token }}
           eslint: true
           eslint_extensions: js,jsx,ts,tsx,json
-          auto_fix: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login == github.repository_owner }}
+          auto_fix: true
+
+      - name: Run eslint
+        if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login == github.repository_owner }}
+        run: yarn eslint .
```
Comment (moodysalem) [inline .github/workflows/lint.yml:52]: the setup-node action sets up problem matchers that should report all the errors output by this job

**Comment by moodysalem** [comment]
to be aware of for prs from forks https://github.com/actions/setup-node/issues/96

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -38,10 +38,15 @@ jobs:
...
         uses: wearerequired/lint-action@36c7e6689e80d785d27a22f71d970f3a3b4fcb70
         with:
           github_token: ${{ secrets.github_token }}
           eslint: true
-          eslint_extensions: js,jsx,ts,tsx,json
-          auto_fix: ${{ github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login == github.repository_owner }}
+          eslint_args: "-c .eslintrc.json"
+          auto_fix: true
+
+      - name: Run eslint
```
Comment (zzmp) [inline .github/workflows/lint.yml:50]: nit: Use the same name for the two lint tasks (eg "Run linters")

File: src/components/ErrorBoundary/index.tsx [hunk 3]
```diff
@@ -126,7 +126,7 @@ function getRelevantState(): null | keyof AppState {
   if (!path.startsWith('#/')) {
     return null
   }
-  const pieces = path.substring(2).split(/[\/\\?]/)
+  const pieces = path.substring(2).split(/[/\\?]/)
```
Comment (zzmp) [inline src/components/ErrorBoundary/index.tsx:129]: Is this change intentional?

File: src/components/LiquidityChartRangeInput/Area.tsx [hunk 4]
```diff
@@ -37,7 +37,7 @@ export const Area = ({
             .y0(yScale(0))(
             series.filter((d) => {
               const value = xScale(xValue(d))
-              return value > 0 && value <= innerWidth
+              return value > 0 && value <= window.innerWidth
```
Comment (zzmp) [inline src/components/LiquidityChartRangeInput/Area.tsx:40]: Is this done by the linter? If not, I think it's better to scope this to the window for mocking.

File: src/hooks/useMonitoringEventCallback.ts [hunk 5]
```diff
@@ -42,7 +42,7 @@ function useMonitoringEventCallback() {
       try {
         push(ref(db, 'trm'), {
           chainId,
-          origin: location.origin,
+          origin: window.location.origin,
```
Comment (zzmp) [inline src/hooks/useMonitoringEventCallback.ts:45]: I like that this is now scoped to `window`.

File: src/state/governance/hooks.ts [hunk 6]
```diff
@@ -136,7 +136,7 @@ function useFormattedProposalCreatedLogs(
 
         // Bravo and one bip proposals omit newlines
         if (startBlock === BRAVO_START_BLOCK || startBlock === ONE_BIP_START_BLOCK) {
-          description = description.replace(/  /g, '\n').replace(/\d\. /g, '\n$&')
+          description = description.replace(/ {2}/g, '\n').replace(/\d\. /g, '\n$&')
```
Comment (zzmp) [inline src/state/governance/hooks.ts:139]: Is this intentional? Is it meant to make it more readable?

**Review by zzmp — APPROVED** [review]
nit: `TextPreset` is a weird name, considering it's just themed text. Would it be difficult to change it to `ThemedText` instead?

File: src/components/ErrorBoundary/index.tsx (hunk 3 above)
Comment (moodysalem) [reply src/components/ErrorBoundary/index.tsx:129]: yeah, linter caught an unnecessary escape

File: src/components/LiquidityChartRangeInput/Area.tsx (hunk 4 above)
Comment (moodysalem) [reply src/components/LiquidityChartRangeInput/Area.tsx:40]: the rule was something like no access to globals without the prefix, this is equivalent to what it was doing before

File: src/hooks/useMonitoringEventCallback.ts (hunk 5 above)
Comment (moodysalem) [reply src/hooks/useMonitoringEventCallback.ts:45]: yeah, linter asked for it

File: src/state/governance/hooks.ts (hunk 6 above)
Comment (moodysalem) [reply src/state/governance/hooks.ts:139]: not intetional, linter autofixed it
